sdk/sphere/Azure.ResourceManager.Sphere/api/Azure.ResourceManager.Sphere.netstandard2.0.cs

```diff
@@ -0,0 +1,611 @@
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Sphere.Models.ProvisioningState left, Azure.ResourceManager.Sphere.Models.ProvisioningState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Sphere.Models.ProvisioningState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Sphere.Models.ProvisioningState left, Azure.ResourceManager.Sphere.Models.ProvisioningState right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct RegionalDataBoundary : System.IEquatable<Azure.ResourceManager.Sphere.Models.RegionalDataBoundary>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public RegionalDataBoundary(string value) { throw null; }
+        public static Azure.ResourceManager.Sphere.Models.RegionalDataBoundary EU { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.RegionalDataBoundary None { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Sphere.Models.RegionalDataBoundary other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Sphere.Models.RegionalDataBoundary left, Azure.ResourceManager.Sphere.Models.RegionalDataBoundary right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Sphere.Models.RegionalDataBoundary (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Sphere.Models.RegionalDataBoundary left, Azure.ResourceManager.Sphere.Models.RegionalDataBoundary right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class SignedCapabilityImageResponse
+    {
+        internal SignedCapabilityImageResponse() { }
+        public string Image { get { throw null; } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct UpdatePolicy : System.IEquatable<Azure.ResourceManager.Sphere.Models.UpdatePolicy>
```

Comment:
This name is quite generic, could we rename it to be more specific?

Reply:
fixed in latest commit